apps/build-onchain-apps/contracts/src/NFTUtilities.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+library NFTUtilities {
```

Comment:
@robpolak @Sneh1999 open question, can this also be part of OnchainKit?

As OnchainKit will have three types of utilities:
- CSS
- TS
- Solidity

Reply:
Not sure this is generic enough to qualify.  Again, I don't want to be authoring solidity components they get widely imported.  This should be referenced as a guide and not a framework to import.

Reply:
Like
```
onchainkit/contracts/utils/
```

Reply:
Given, I don't think we want to take on that scope of work & we don't have the resources to audit to the depth I think we should.

Reply:
Maybe I need to pull this folder https://github.com/coinbase/build-onchain-apps/tree/main/apps/build-onchain-apps/src/onchain a bit up, as we can reuse it for other things as well. 🤔

Reply:
yea, I also dont think we should